### ISSUE TITLE
BatchNormalization Layer also fits for RNN/LSTM ?

### ISSUE THREAD

**Issue by wenruij** [fchollet/keras#3730]
hi @bmabey 

   Just curious, does the BatchNormalization Layer in Keras also fit for RNN/LSTM ?

  Seems the references for BatchNormalization Layer do NOT mention the [Batch Normalized Recurrent Neural Networks](https://arxiv.org/abs/1510.01378)

  Thanks
